thorin/pass/pipelinebuilder.h

```diff
@@ -5,35 +5,90 @@
 
 #include "thorin/pass/optimize.h"
 #include "thorin/pass/pass.h"
+#include "thorin/phase/phase.h"
 
 namespace thorin {
 
```

Comment:
can we switch to newer C++ with `using` instead of `typedef`? We are now consistently using `using` everywhere and also `using` formats nicecly with `clang-format`.